legal-api/report-templates/certificateOfAmalgamation.html

```diff
@@ -39,7 +39,7 @@
                     <div class="cert-separator"></div>
                     <div class="cert-issued-by">
                         <div>Issued under my hand at Victoria, British Columbia</div>
-                        <div>on {{ report_date_time }}</div>
+                        <div>on {{ report_date }}</div>
```

Comment:
This is not the same as [Certificate of Incorporation](https://github.com/bcgov/lear/blob/47dc70f37c6e44f11b86ba6a733c60da83fec5ee/legal-api/report-templates/certificateOfIncorporation.html#L36), but then several variables in here are different so maybe this is OK. I don't know.

Reply:
James, can you please test this on a past amalgamation and verify that this date is the same as the effective date (not today's date)? Please post results here. Thanks.

Reply:
`report_date` will print current date. thats not what Certificate of IA does. replace `report_date` with `business.formatted_founding_date`

Reply:
Confirmed with Yui. I.e. the date of amalgamation is Feb 16, and the certificate is generated today, the issuance date should be Feb 21 on a date(not datetime) format.

Reply:
Certificate of Incorporation has business.formatted_founding_date

Reply:
if its wrong then we need to revisit other certificates as well

Reply:
> Certificate of Incorporation has business.formatted_founding_date

Why is that different from what Yui said?

Reply:
will check with Yui

Reply:
Yui confirmed we need to follow IA. please update as i mentioned above

Reply:
certificates which shows `report_date` might be a bug. Yui will follow up with Naveen on that

Reply:
yes. just follow Certificate of IA

Reply:
@jamespaologarcia How about a screenshot of a Certificate of Amalgamation that was done before today? Thanks.